spec/Subject-spec.js

```diff
@@ -508,4 +508,77 @@ describe('Subject', function () {
 
     source.subscribe(subject);
   });
+
+  describe('asObservable', function () {
+    it('should hide subject', function () {
+      var subject = new Rx.Subject();
+      expect(subject).not.toEqual(subject.asObservable());
+    });
```

Comment:
Seems like there should be a test like `expect(result instanceof Observable).toBe(true)` and `expect(result instanceof Subject).toBe(false)`